File: MHT_matlab/selectAppFeat.m
function [cur_observation other_param] = selectAppFeat(cur_observation, other_param)

switch other_param.appSel
    case 'cnn'
        cur_observation.app = cur_observation.cnn;
        other_param.isAppModel = 1;
    otherwise
        cur_observation.app = [];
        other_param.isAppModel = 0;
end
       
end

File: MHT_python/selectAppFeat.py


def selectAppFeat(det, other_param):
    if(other_param['appSel'] == 'cnn'):
        det['app'] = det['cnn']
        other_param['isAppModel'] = 1
    else:
        det['app'] = []
        other_param['isAppModel'] = 0
    print('Selected the App Feat\n')
    return det, other_param
